src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecordUtils.java

```diff
@@ -372,12 +385,18 @@ public static SVCallRecord create(final VariantContext variant, boolean keepVari
             }
         } else {
             contigB = contigA;
-            positionB = variant.getEnd();
+            // Force reset of END coordinate
+            if (type.equals(StructuralVariantType.INS)) {
+                positionB = positionA + 1;
```

Comment:
Yes that's true, but so far the GATK tools we are using in the pipeline would not affect this where it matters (GenerateBatchMetrics through GenotypeBatch). 

In the future, however, I'd like to use POS/END to mark the nominal location of the insertion but separate INFO tags to indicate the left/right SR coordinates. This will avoid difficulties with making sure POS precedes END, which really should not be used for marking the SR positions IMO according to the VCF spec.

Reply:
Totally agreed, we should have separate info fields for SR positions.